test/ci-operator-integration/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Please quote everything to avoid ugly errors and also so we can use shellcheck to validate these files:

```
$ shellcheck test/ci-operator-integration/run.sh

In test/ci-operator-integration/run.sh line 4:
   jq --argfile a $1 -n 'def post_recurse(f): def r: (f | select(. != null) | r), .; r; def post_recurse: post_recurse(.[]?); ($a | (post_recurse | arrays) |= sort) as $a | $a' > $2
                  ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                                                                                                                                                   ^-- SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 31:
if ! ci-operator --dry-run --namespace ${TEST_NAMESPACE} --config ${TEST_CONFIG} > ${DRY_RUN_JSON} 2>/dev/null; then
                                                                  ^------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                   ^-------------^ SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 37:
sort_nested_JSON_to_file ${DRY_RUN_JSON} ${DRY_RUN_JSON_SORTED}
                         ^-------------^ SC2086: Double quote to prevent globbing and word splitting.
                                         ^--------------------^ SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 38:
if ! diff -b -I "[0-9a-f]\{40\}" -I "[0-9a-f]\{64\}" ${EXPECTED} ${DRY_RUN_JSON_SORTED}; then
                                                     ^---------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                 ^--------------------^ SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 50:
if ! ci-operator --dry-run --namespace ${TEST_NAMESPACE} --config ${TEST_CONFIG} --template ${TEST_TEMPLATE} --target test-template --artifact-dir ${ARTIFACT_DIR} > ${DRY_RUN_WITH_TEMPLATE_JSON} 2>/dev/null; then
                                                                  ^------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                            ^--------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                                                                                   ^-------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                                                                                                     ^---------------------------^ SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 55:
sort_nested_JSON_to_file ${DRY_RUN_WITH_TEMPLATE_JSON} ${DRY_RUN_WITH_TEMPLATE_JSON_SORTED}
                         ^---------------------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                       ^-- SC2086: Double quote to prevent globbing and word splitting.


In test/ci-operator-integration/run.sh line 56:
if ! diff -b -I "[0-9a-f]\{40\}" -I "[0-9a-f]\{64\}" ${EXPECTED_WITH_TEMPLATE} ${DRY_RUN_WITH_TEMPLATE_JSON_SORTED}; then
                                                     ^-----------------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                               ^-- SC2086: Double quote to prevent globbing and word splitting.

For more information:
```

Reply:
+1

Reply:
addressed